repos/system_upgrade/el8toel9/actors/checkblsgrubcfgonppc64/libraries/blsgrubcfgonppc64.py

```diff
@@ -25,14 +31,16 @@ def process():
                 'Leapp cannot continue with upgrade on "ppc64le" bare metal systems'
             ),
             reporting.Summary(
-                'Currently, there is a known issue that prevents Leapp from upgrading '
-                'ppc64le bare metal systems. You can file a bug in http://bugzilla.redhat.com/ '
-                'for leapp-repository component and include "[ppc64le bare metal upgrade]" in '
-                'the title to get the issue prioritized.'
+                'In-place upgrade to RHEL 9 is not supported on POWER8 and POWER9 bare metal systems. '
+                'For more information, please refer to the linked KB article.'
```

Comment:
let's put the link here yet, so it's printed in the txt. until we start to print external URLs in the txt file